README.adoc

```diff
@@ -215,11 +196,23 @@ Arquillian tests in which servlets are created to perform the in-container testi
 == Running the tests
 
 It's now time to build and run your Arquillian tests.
-Run the Maven build command in the `start` directory.
+Navigate to the `start` directory. First, run the Maven command to package the application.
+Then, run the `liberty-maven-plugin` goals to create the application server, install the features
+and deploy the application to the server. The `configure-arquillian` goal configures
+your Arquillian container.
```

Comment:
I'd put a colon after "The `configure-arquillian` goal configures your Arquillian container" (instead of a period) to lead into the examples that follow.